webpay/controllers/front/webpaypluspaymentvalidate.php

```diff
@@ -31,31 +31,37 @@ public function initContent()
         //3. Pago abortado (con botón anular compra en el formulario de Webpay): llegan TBK_TOKEN, TBK_ID_SESION, TBK_ORDEN_COMPRA
         //4. Caso atipico: llega todos token_ws, TBK_TOKEN, TBK_ID_SESION, TBK_ORDEN_COMPRA
         $params = $_SERVER['REQUEST_METHOD'] === 'POST' ? $_POST : $_GET;
-        $tokenWs = isset($params['token_ws']) && $params['token_ws']!==null ? $params['token_ws'] : null;
-        $tbktoken = isset($params['TBK_TOKEN']) && $params['TBK_TOKEN']!==null ? $params['TBK_TOKEN'] : null;
-        $tbkIdSesion = isset($params['TBK_ID_SESION'])&& $params['TBK_ID_SESION']!==null ? $params['TBK_ID_SESION'] : null;
-        if($this->isDebugActive()){
+        $tokenWs = $params['token_ws'] ?? null;
+        $tbktoken = $params['TBK_TOKEN'] ?? null;
+        $tbkIdSesion = $params['TBK_ID_SESION'] ?? null;
+        if ($this->isDebugActive()) {
             $this->logInfo("C.1. Iniciando validación luego de redirección desde tbk =>
                 method: {$_SERVER['REQUEST_METHOD']}");
             $this->logInfo(json_encode($params));
         }
 
-        if (isset($tokenWs) && !isset($tbktoken)) {//Flujo 1 => Confirmar Transacción
-
+        if (isset($tokenWs)) {
```

Comment:
What happens if the token is in the other variables?

Reply:
Only come in another variable in cases of abandonment or timeout